.github/workflows/changeset-reporter.yml

```diff
@@ -0,0 +1,50 @@
+name: changeset-reporter
+
+# This workflow adds a comment to the PR depending on the results of the pr-check-changeset workflow.
+#
+# The workflows are separated for security reasons. This workflow requires write access to the repo, but the
+# pr-check-changeset workflow does not.
+
+on:
+  workflow_run:
+    workflows: [ "pr-check-changeset" ]
+    types:
+      - completed
+
+jobs:
+  load_report:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download results
+        uses: dawidd6/action-download-artifact@bd10f381a96414ce2b13a11bfa89902ba7cea07f # ratchet:dawidd6/action-download-artifact@v2.24.3
```

Comment:
There's no official action to download an artifact like there is for publishing them?

Reply:
I know, it's a little unexpected. You can only upload/download artifacts in the same workflow as far as I can tell.

"Let's suppose you have a workflow with a job in it that at the end uploads an artifact using actions/upload-artifact action and you want to download this artifact in another workflow that is run after the first one. Official actions/download-artifact does not allow this."